downstream/modules/platform/proc-controller-add-source.adoc

```diff
@@ -52,7 +53,7 @@ If the key is not found, then the host is enabled.
 
 * *Overwrite*: If checked, any hosts and groups that were previously present on the external source but are now removed, are removed from
 the {ControllerName} inventory.
-Hosts and groups that were not managed by the inventory source are promoted to the next manually created group, or, if there is no manually created group to promote them into, they are left in the "all" default group for the inventory.
+Hosts and groups that were not managed by the inventory source are promoted to the next manually created group, or, if there is no manually created group to promote them into, they are left Select the inventory name in the "all" default group for the inventory.
```

Comment:
looks a bit odd: 'they are left *Select the inventory name* in the "all" default group...